frontend/components/ProductCard.jsx

```diff
@@ -18,7 +21,59 @@ function ProductCard_(props, ref) {
   //
   // By default, we are just piping all ProductCardProps here, but feel free
   // to do whatever works for you.
-  return <PlasmicProductCard root={{ ref }} {...props} />;
+
+  const { tool } = props;
+
+  const [learnMoreActive, setLearnMoreActive] = useState(false);
+
+  const commonProps = {
+    root: { ref },
+    logo: tool.logo_image
+  };
+  if (!learnMoreActive) {
+    return (
+      <PlasmicProductCard
+        {...commonProps}
+        withoutScreenshotButtons={true}
+        title={tool.name}
+        description={<span dangerouslySetInnerHTML={{ __html: tool.short_description }} />}
+        learnMoreSlot={
+          <TitleLearnMoreButton
+            onClick={(e) => {
+              e.preventDefault();
+              setLearnMoreActive(true)
+            }}
+          >
+            {tool.name}
```

Comment:
I find it kinda confusing that that we are using the name of the tool for the button. Was that something recommended by the UX team? I'd probably add a header for this variant with the tool name and then keep this as "Find Out More"

Reply:
We've been back and forth about this button. Looking at the latest UX review, this needs to be changed but it could be done in a new issue.

Change the blue button text
* Currently, the idea was to change it to “Find Out More,” with screen reader text saying “Find out more about X”
* One other suggestion from the DUX team is “More Details” with screen reader text saying “More X Details”
* Overall, this makes for more succinct button text

We need to test how the screenreader reads the content.